docs/docs/tutorials/ssh-signing.md

```diff
@@ -0,0 +1,103 @@
+# :material-key: SSH and Commit Signing
+
+This tutorial will guide you through setting up SSH keys and commit signing for GitHub. SSH keys are a more secure way to connect to GitHub, rather than other protocols like HTTPS. Commit signing is used to verify that commits are coming from you and have not been tampered with.
+
+!!! warning
+    You must have a verified email address on GitHub to add SSH keys and sign commits.
+
+## Create an SSH Key
+
+=== "Windows"
+
+    In an admin PowerShell terminal, enter the following command:
+
+    ```text
+    ssh-keygen -t ed25519 -C "your_github_email@example.com"
+    ```
+
+    Save to the default location `c:/Users/YOU/.ssh/id_ed25519` and do not enter a passphrase. This will allow you to use the key without entering a password every time you push to GitHub.
```

Comment:
start the sentence with "When prompted, save...". I didn't realize, until running the command, that this command prompts for a file location, so I was expecting to need to copy the output manually